vite.config.js

```diff
@@ -18,10 +18,12 @@ const fontImport = ViteFonts({
 // https://vitejs.dev/config/
 export default ({ command, mode }) => {
   const scalaClassesDir = path.resolve(__dirname, "explore/target/scala-2.13");
+  const isProduction = mode == "production";
   const sjs =
-    mode == "production"
+    isProduction
       ? path.resolve(scalaClassesDir, "explore-opt")
       : path.resolve(scalaClassesDir, "explore-fastopt");
+  const rollupPlugins = isProduction ? [] : [visualizer()];
```

Comment:
@rpiaggio Here we select that in production we don't run the visualizer

Reply:
Missed that, thank you! 👍